config/audit.yaml

```diff
@@ -12,10 +12,12 @@ audit:  # Configurations for auditing
     population:
       attack_data_fraction: 0.1  # Fraction of the auxilary dataset to use for this attack
     lira:
-      training_data_fraction: 0.1  # Fraction of the auxilary dataset to use for this attack (in each shadow model training)
-      num_shadow_models: 8 # Number of shadow models to train
-      online: False # perform online or offline attack
-      fixed_variance: True # Use a fixed variance for the whole audit
+      training_data_fraction: 0.5  # Fraction of the auxilary dataset to use for this attack (in each shadow model training)
+      num_shadow_models: 16 # Number of shadow models to train
+      exclude_logit_threshold: 1 # Set a threshold for when to skip a logit based (Useful for unbalance in IN vs OUT samples)
+      online: True # perform online or offline attack
+      memorization: True # Use memorization score to boost performance
+      memorization_threshold: 0.99 # The most vulnerable percentile ( 0.0 for original threshold )
```

Comment:
So we do not have an automatic way to do this now? Did you write an issue?

Reply:
We do! For memorization alone there is 5 possible flags that can be used. Only memorization True/False is left in audit.yaml now. A question is what kind of flags do we wanna set/ should be visible  in the audit.yaml file. Should have some sort of standard for this?